Transaction/index.ts

```diff
@@ -73,6 +74,7 @@ export namespace Transaction {
 		flags: isly.array(isly.string() || "review"),
 		oldFlags: isly.string().array(),
 		notes: Note.type.array(),
+		risk: isly.number().optional(),
```

Comment:
Is there a reasonable way to make this only pass integers?

Reply:
yes, but is that what we want?

Reply:
\+